### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by noffle**
formatting and info love :heart:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,43 @@
...
 GLOBAL OPTIONS:
 	--help, -h		show help
-	--version, -v	print the version
+	--version, -v		print the version
 ```
 
+### Install
 
+```
+go install https://github.com/whyrusleeping/iptb
```
Comment (whyrusleeping) [inline README.md]: `go get github.com/whyrusleeping/iptb`

File: README.md [hunk 2]
```diff
@@ -1,38 +1,43 @@
...
-computer. It allows the creation of up to 1000 or more nodes! Allows for
-various setup options to be selected such as different bootstrapping patterns.
-iptb makes testing networks in ipfs easy!
+`iptb` is a program used to create and manage a cluster of sandboxed IPFS nodes locally on your computer. Spin up 1000s of nodes! It exposes various options, such as different bootstrapping patterns. `iptb` makes testing IPFS networks easy!
 
 ### Usage:
 ```
 NAME:
-	iptb - A new cli application
+	iptb - The IPFS TestBed
```
Comment (whyrusleeping) [inline README.md]: could probably drop the name part here. its just lazy copy-paste

Comment (noffle) [inline README.md]: Made this more explicit -- I think having the full command output is useful (consistent /w what they'll see when they run --help themselves)

File: README.md (hunk 1 above)
Comment (noffle) [inline README.md]: whoops -- done!

**Comment by noffle** [comment]
ready!

File: README.md [hunk 3]
```diff
@@ -1,38 +1,45 @@
...
 GLOBAL OPTIONS:
 	--help, -h		show help
-	--version, -v	print the version
+	--version, -v		print the version
 ```
 
+### Install
 
+```
+go get https://github.com/whyrusleeping/iptb
```
Comment (whyrusleeping) [inline README.md]: remove the https

**Comment by whyrusleeping** [comment]
so close!

**Comment by noffle** [comment]
done!

**Pull request merged**